library/Booster/UnsafeGlobalState.hs

```diff
@@ -0,0 +1,33 @@
+module Booster.UnsafeGlobalState (
+    unsafeReadGlobalMaxIterations,
+    unsafeReadGlobalMaxRecursion,
+    writeGlobalMaxIterations,
+    writeGlobalMaxRecursion,
+) where
+
+import Data.IORef
+import System.IO.Unsafe (unsafePerformIO)
+
+import Booster.Util (Bound (..))
+
+{-# NOINLINE globalMaxIterations #-}
+globalMaxIterations :: IORef (Bound "Iterations")
+globalMaxIterations = unsafePerformIO (newIORef 100)
+
+{-# NOINLINE unsafeReadGlobalMaxIterations #-}
+unsafeReadGlobalMaxIterations :: Bound "Iterations"
+unsafeReadGlobalMaxIterations = unsafePerformIO $ readIORef globalMaxIterations
+
+writeGlobalMaxIterations :: Bound "Iterations" -> IO ()
+writeGlobalMaxIterations = atomicWriteIORef globalMaxIterations
+
+{-# NOINLINE globalMaxRecursion #-}
+globalMaxRecursion :: IORef (Bound "Recursion")
+globalMaxRecursion = unsafePerformIO (newIORef 5)
```

Comment:
Should we maybe use _a single `IORef`_  holding`EquationOptions` ?

o/t https://drive.google.com/drive/folders/1YkPxlz862FhttZ5uVRC3z9FN5Ax7wWkX?usp=sharing

Reply:
Sounds good! Done.